app/javascript/components/users/user/AccountInfo.jsx

```diff
@@ -25,7 +25,11 @@ AccountInfo.propTypes = {
     name: PropTypes.string.isRequired,
     email: PropTypes.string.isRequired,
     provider: PropTypes.string.isRequired,
-    role: PropTypes.string.isRequired,
+    role: PropTypes.shape({
```

Comment:
I know this is irrelevant to this PR - In the future, we may extract redundant PropTypes into helper files where we can easily import them on need,